google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/cloud/CloudSdkAppEngineHelperTest.java

```diff
@@ -140,11 +150,13 @@ public void testCreateDeployRunnerInvalidSdk() {
 
     assertNull(runner);
     verify(callback, times(1))
-        .errorOccurred("No Cloud SDK was found in the specified directory. /this/path");
+        .errorOccurred(
+            "No Cloud SDK was found in the specified directory. " + path.toString());
```

Comment:
Unfortunately in this case the path in the verification is incorrect on windows.

At a higher level, I think it's painful to verify exact messages.  Ideally we'd be verifying that a certain type of error occurred.